x/programs/runtime/import_log_debug.go

```diff
@@ -3,6 +3,11 @@
 
 //go:build debug
 
+/*
+ * Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+ * See the file LICENSE for licensing terms.
+ */
+
```

Comment:
Is this right here? Feels like we should be replacing the blob at the top of the file here